keras/src/models/functional_test.py

```diff
@@ -5,6 +5,7 @@
 import pytest
 from absl.testing import parameterized
 
+import keras
```

Comment:
Oh `keras` is often not imported in unit tests to not import everything and to avoid circular references.

In this case you can just do `from keras.src.models import model_from_json` and remove this.

Reply:
Got it, fixed